test/unit-test/vp8/vp8_utest.c

```diff
@@ -301,11 +303,168 @@ void test_VP8_Packetizer_TL0PICIDX( void )
 
     result = VP8Packetizer_Init( &( ctx ),
                                  &( frame ) );
+
+    TEST_ASSERT_EQUAL( VP8_RESULT_OK,
+                       result );
+
+    pkt.pPacketData = &( packetBuffer[ 0 ] );
+    pkt.packetDataLength = PACKET_BUFFER_LENGTH;
+
+    result = VP8Packetizer_GetPacket( &( ctx ),
+                                      &( pkt ) );
+
+    while( result != VP8_RESULT_NO_MORE_PACKETS )
```

Comment:
Can we calculate the number of packet first? Then use the calculated number to check the expected result. For example, if we expect to have 3 packets from this frame, we can expect to have 3 OK returned and 1 NO_MORE_PACKETS at the end.